lib/todo-list-view.coffee

```diff
@@ -151,11 +156,11 @@ class TodoListView extends ScrollView
     if atom.config.get('todo-list.showOnRightSide')
       @element.classList.remove('panel-left')
       @element.classList.add('panel-right')
-      atom.workspaceView.appendToRight(this)
+      atom.workspace.addRightPanel(item: this)
     else
       @element.classList.remove('panel-right')
       @element.classList.add('panel-left')
-      atom.workspaceView.appendToLeft(this)
+      atom.workspace.addLeftPanel(item: this)
```

Comment:
Same here I'm not 100% sure this is a correct upgrade path but I know for sure `appendToLeft` no longer exists.